src/test/java/Homework17.java

```diff
@@ -1,30 +1,23 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 public class Homework17 extends BaseTest {
 
     @Test
-    public void addSongToPlaylist() throws InterruptedException {
-        login("demo@class.com", "te$t$tudent");
-        Thread.sleep(2000);
-        grabASong();
-
-    }
-
-    public void grabASong(){
-        WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
-        WebElement playlist = driver.findElement(By.xpath("//section[@id='playlists']//li[5]"));
-
-        Actions acts = new Actions(driver);
-        acts.clickAndHold(song)
-                .release(playlist)
-                .build()
-                .perform();
-
-        playlist.click();
-//        acts.dragAndDrop(song, playlist).build().perform();
-
-    }
+        public void addSongtoPlaylist() throws InterruptedException {
+        String newSongAddedNotification = "Added 1 song into";
+        navigateToPage();
+        provideEmail("zyabruk@gmail.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+        searchSong("Pluto");
+        viewAllSearchResults();
+        selectFirstSongResult();
+        clickAddToButton();
+        choosePlaylist("Test Pro Playlist");
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/213385444-ec7f01e4-676a-4424-a72f-8e32f69355b4.png)

![image](https://user-images.githubusercontent.com/104761482/213385697-795997da-68a3-40fe-ac9c-2f09cef494ef.png)


![image](https://user-images.githubusercontent.com/104761482/213385627-8f9b8d17-1b44-4ff1-975c-bfae934a0aee.png)